cypress.json

```diff
@@ -3,5 +3,10 @@
     "projectId": "sc56ms",
     "testFiles": "**/*.feature",
     "experimentalInteractiveRunEvents": true,
-    "video": false
+    "video": false,
+    "env": {
+        "dhis2DataTestPrefix": "dhis2-scheduler",
+        "networkMode": "live",
+        "dhis2ApiVersion": "37"
```

Comment:
Should this not also be 38?

Reply:
Yeah, I would think so as well... Hendrik added this, but I guess the param from the command takes precedence, but will fix.